### DISCUSSION TITLE
Custom login layout settings based on session selected?

### DISCUSSION THREAD

**Discussion by d3al** [neutrinolabs/xrdp#2741] [Q&A]
I have multiple sessions and one session does not require credentials and the other one does. The dialog layout is setup for credential fields in mind so there is a large area of dialog that is empty space.

I would like to have the dialog, buttons and even if possible the graphic change positions/size based off the session selected, is this possible and how?

**Comment by matt335672** [comment]
No it isn't. The dialog box is currently a fixed size.
